### PR TITLE
8212369.WPR version α0.01

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/102171)

### CODE REVIEW

File: manifests/8/8212369/WPR/α0.01/8212369.WPR.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: 8212369.WPR
+PackageVersion: α0.01
+PackageLocale: en-US
+Publisher: 8212369
+PackageName: WPR
+License: Unknown
```
Comment (Trenly) [inline manifests/8/8212369/WPR/α0.01/8212369.WPR.locale.en-US.yaml:9]:
```suggestion
License: MS-PL
```
https://github.com/8212369/WPR/blob/master/UI/WPR.UI/Pages/AboutPage.axaml

File: manifests/8/8212369/WPR/α0.01/8212369.WPR.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: 8212369.WPR
+PackageVersion: α0.01
+Installers:
+- InstallerUrl: https://github.com/8212369/WPR/releases/download/0.0.1-alpha/wpr_release_desktop_alpha_001.zip
+  Architecture: x64
+  InstallerType: zip
+  NestedInstallerType: exe
```
Comment (Trenly) [inline manifests/8/8212369/WPR/α0.01/8212369.WPR.installer.yaml:10]:
```suggestion
  NestedInstallerType: portable
```

File: manifests/8/8212369/WPR/α0.01/8212369.WPR.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: 8212369.WPR
+PackageVersion: α0.01
```
Comment (Trenly) [inline manifests/8/8212369/WPR/α0.01/8212369.WPR.installer.yaml:5]:
```suggestion
PackageVersion: 0.0.1-alpha
```
Prefixes on versions do not work well, and unicode characters should be avoided as they do not work well with scripts in many cases

File: manifests/8/8212369/WPR/α0.01/8212369.WPR.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: 8212369.WPR
+PackageVersion: α0.01
```
Comment (Trenly) [inline manifests/8/8212369/WPR/α0.01/8212369.WPR.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 0.0.1-alpha
```

File: manifests/8/8212369/WPR/α0.01/8212369.WPR.yaml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: 8212369.WPR
+PackageVersion: α0.01
```
Comment (Trenly) [inline manifests/8/8212369/WPR/α0.01/8212369.WPR.yaml:5]:
```suggestion
PackageVersion: 0.0.1-alpha
```

**Review by Trenly — CHANGES_REQUESTED** [review]
Package install is not silent, package has dependencies on .NET 5.0.17

image

**Comment by upintheairsheep** [comment]
I seriously wish Microsoft allowed you to rename folders in the web interface.

**Comment by upintheairsheep** [comment]
Revert

**Comment by upintheairsheep** [comment]
Can you move the file’s appropriately in GitHub desktop?

**Comment by stephengillie** [comment]
> Can you move the file’s appropriately in GitHub desktop?

They should be moved in the file system, within a branch of a fork, and GitHub Desktop or another Git client used to commit and push these changes back to GitHub. I do not believe GitHub Desktop has file manager functionality, as most Git clients that I have seen do not.

**Comment by ItzLevvie** [comment]
[Policy] Dependencies

**Comment by upintheairsheep** [comment]
> [Policy] Dependencies

The program does not appear to need them, but just try it out to see if it does

**Comment by upintheairsheep** [comment]
Do the thing, by renaming

**Comment by upintheairsheep** [comment]
Hi

**Comment by stephengillie** [comment]
> Package install is not silent, package has dependencies on .NET 5.0.17
> 
> image

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 
> Faulting application name: WPR.UI.Desktop.exe, version: 1.0.0.0, time stamp: 0x62577b55 
> Faulting module name: KERNELBASE.dll, version: 10.0.19041.1202, time stamp: 0xc9db1934 Exception code: 0xe0434352 Fault offset: 0x0000000000034f99 
> Faulting process id: 0x1204 
> Faulting application start time: 0x01da3e9f4ae297d2 
> Faulting application path: C:\Users\User\AppData\Local\Microsoft\WinGet\Packages\8212369.WPR__DefaultSource\WPR.UI.Desktop.exe 
> Faulting module path: C:\Windows\System32\KERNELBASE.dll Report Id: e374d6b1-fd01-433c-89a6-476d045d4b0f 
> Faulting package full name:  
> Faulting package-relative application ID:   Application: WPR.UI.Desktop.exe CoreCLR Version: 5.0.1722.21314 .NET Version: 5.0.17 Description: The process was terminated due to an unhandled exception. Exception Info: System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\Windows\system32\Database\TrueAchievements'.    at System.IO.Enumeration.FileSystemEnumerator`1.CreateDirectoryHandle(String path, Boolean ignoreNotFound)    at System.IO.Enumeration.FileSystemEnumerator`1.Init()    at System.IO.Enumeration.FileSystemEnumerator`1..ctor(String directory, Boolean isNormalized, EnumerationOptions options)    at System.IO.Enumeration.FileSystemEnumerable`1..ctor(String directory, FindTransform transform, EnumerationOptions options, Boolean isNormalized)    at System.IO.Enumeration.FileSystemEnumerableFactory.UserDirectories(String directory, String expression, EnumerationOptions options)    at System.IO.Directory.InternalEnumeratePaths(String path, String searchPattern, SearchTarget searchTarget, EnumerationOptions options)    at WPR.Common.Filesystem.CopyFilesRecursively(String sourcePath, String targetPath)    at WPR.UI.Desktop.Program.Main(String[] args) 

(Automated response - build 513.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 
> Faulting application name: WPR.UI.Desktop.exe, version: 1.0.0.0, time stamp: 0x62577b55 
> Faulting module name: KERNELBASE.dll, version: 10.0.19041.1202, time stamp: 0xc9db1934 Exception code: 0xe0434352 Fault offset: 0x0000000000034f99 
> Faulting process id: 0x8d8 
> Faulting application start time: 0x01da3ea2105b7aa3 
> Faulting application path: C:\Users\User\AppData\Local\Microsoft\WinGet\Packages\8212369.WPR__DefaultSource\WPR.UI.Desktop.exe 
> Faulting module path: C:\Windows\System32\KERNELBASE.dll Report Id: b7fd6e6c-e35f-43db-86dc-f11cc7d771d9 
> Faulting package full name:  
> Faulting package-relative application ID:   Application: WPR.UI.Desktop.exe CoreCLR Version: 5.0.1722.21314 .NET Version: 5.0.17 Description: The process was terminated due to an unhandled exception. Exception Info: System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\Windows\system32\Database\TrueAchievements'.    at System.IO.Enumeration.FileSystemEnumerator`1.CreateDirectoryHandle(String path, Boolean ignoreNotFound)    at System.IO.Enumeration.FileSystemEnumerator`1.Init()    at System.IO.Enumeration.FileSystemEnumerator`1..ctor(String directory, Boolean isNormalized, EnumerationOptions options)    at System.IO.Enumeration.FileSystemEnumerable`1..ctor(String directory, FindTransform transform, EnumerationOptions options, Boolean isNormalized)    at System.IO.Enumeration.FileSystemEnumerableFactory.UserDirectories(String directory, String expression, EnumerationOptions options)    at System.IO.Directory.InternalEnumeratePaths(String path, String searchPattern, SearchTarget searchTarget, EnumerationOptions options)    at WPR.Common.Filesystem.CopyFilesRecursively(String sourcePath, String targetPath)    at WPR.UI.Desktop.Program.Main(String[] args) 

(Automated response - build 513.)

**Comment by stephengillie** [comment]
>  Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\8\8212369\WPR\α0.01', expected 'manifests\8\8212369\WPR\0.0.1-alpha'

File: manifests/8/8212369/WPR/0.0.1-alpha/8212369.WPR.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: 8212369.WPR
+PackageVersion: 0.0.1-alpha
```
Comment (stephengillie) [inline manifests/8/8212369/WPR/0.0.1-alpha/8212369.WPR.installer.yaml:5]:
```suggestion
PackageVersion: 0.0.1-alpha
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.5
```

**Pull request closed without merging**